src/AWSMetadata.jl

```diff
@@ -100,9 +100,10 @@ function _generate_high_level_wrapper(services::Array{OrderedDict})
         service_path = joinpath(@__DIR__, "services", "$service_name.jl")
         open(service_path, "w") do f
             println(f, "# This file is auto-generated by AWSMetadata.jl")
-            println(f, "include(\"../AWSServices.jl\")")
-            println(f, "using Compat")
+            println(f, "using AWS")
             println(f, "using .AWSServices: $service_name\n")
```

Comment:
```suggestion
            println(f, "using AWS.AWSServices: $service_name\n")
```
in case of shadowing